### PR TITLE
Added new course: 21254

### PR DESCRIPTION
**Pull request by CasperWong-jpg**
# Summary of Changes

- Please indicate the classes you are modifying:
  - 21-254 (new)
  - 18-202 (Hyperlink)
  - README (Hyperlink)

# Checklist:

- [Y] I have added my entries to the `README.md` table of contents in the root directory as links.

Make sure that your contributions:

- [To the best of my knowledge] Are accurate and will not mislead the reader
- [Y] Do not encourage or cause academic integrity violations
- [Y] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ expect from the core classes from the ECE and CS programs at CMU.
 - [36-219: Probability Theory and Random Processes](math_science_reqs/36219.md)
 - [36-225: Introduction to Probability Theory](math_science_reqs/36225.md)
 - [21-259: Calculus in Three Dimensions](math_science_reqs/21259.md)
+- [21-254: Linear Algebra and Vector Calculus for Engineers]((math_science_reqs/25254.md))
```
Comment (mikinty) [inline README.md]: Typo?

```suggestion
- [21-254: Linear Algebra and Vector Calculus for Engineers]((math_science_reqs/21254.md))
```

File: math_science_reqs/18202.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ final grade)
 The Mathematical Foundations requirement can also be fulfilled by taking 2 of the following instead:
 - 21-259: Calculus in Three Dimensions
 - [21-241: Matrices and Linear Transformations](math_science_reqs/21241.md)
-- 21-254: Linear Algebra and Vector Calculus for Engineers
+- [21-254: Linear Algebra and Vector Calculus for Engineers ](math_science_reqs/25254.md)
```
Comment (mikinty) [inline math_science_reqs/18202.md]: Change the link here

File: math_science_reqs/24254.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# 21-254: Linear Algebra and Vector Calculus for Engineers
```
Comment (mikinty) [inline math_science_reqs/24254.md]: File is named incorrect

File: math_science_reqs/24254.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+| Category | Difficulty |
+| :------: | :--------: |
+|   HW     |     4      |
+|  Exams   |     4      |
+
+Linear Algebra and Vector Calculus for Engineers is a fast-moving class that covers a broad range of topics. It essentially packs the content of two courses into a single semester: the first half is linear algebra, such as computing matrices and eigenvalues (which is also a topic covered in 18202), then vector calculus (which is similar to the second half of 21259). Due to its breadth, it does cover material at much less depth, which you may or may not prefer. The course definitely seems to be oriented more for engineering, skipping over proofs and justifications and focusing more on calculations and using Matlab.
```
Comment (mikinty) [inline math_science_reqs/24254.md]: Do you mind linking the classes here to their docs? 18202 and 21259?

**Review by mikinty — CHANGES_REQUESTED** [review]
Can you add some links and fix the file name?

File: math_science_reqs/24254.md (hunk 4 above)
Comment (CasperWong-jpg) [reply math_science_reqs/24254.md]: All done, thanks for the bug catches!

**Review by mikinty — APPROVED** [review]
Thank you for making changes

**Pull request merged**